novaagent/libs/centos.py

```diff
@@ -168,11 +166,14 @@ def _os_defaults_network_manager():
         :rtype: bool
         :return: has network manager only, not network scripts
         """
-        dist = distro.name()
+        dist = distro.id()
+        server_os_version = ServerOS.version_tuple()
 
-        if dist in ['rhel', 'centos'] and ServerOS.version_tuple() >= (7, 6):
+        log.info("Linux Distribution Detected: {0} Version {1}".format(
+            dist, ServerOS.version_tuple()))
+        if dist in ['rhel', 'centos'] and server_os_version >= (8,):
```

Comment:
This doesn't work on RHEL 8 beta.  `version_tuple` comes back as `(8, 0)`, and `(8,) >= (8,0)` evaluates as False because when comparing tuples, if they are different lengths and the common parts are equal, the longer tuple is greater than the shorter tuple.  For the logic to work correctly it needs to be this:

```suggestion
        if dist in ['rhel', 'centos'] and server_os_version >= (8, 0):
```